test/emqx_acl_cache_SUITE.erl

```diff
@@ -55,6 +55,27 @@ t_clean_acl_cache(_) ->
     ?assertEqual(0, length(gen_server:call(ClientPid, list_acl_cache))),
     emqtt:stop(Client).
 
+
+t_discard_acl_cache(_) ->
```

Comment:
i have no idea if that kind of a test is sufficient. any feedback appreciated

Reply:
This should be enough IMO.
BTW this is not a `unit` test (as you stated in commit message).
In Erlang, unit is `eunit` (which are typically implemented in `_tests.erl` modules, or locally in the module under test).
Test cases in `_SUITE.erl` modules are `common` tests, or `ct`

Reply:
ahh, good to know ;-)